### PR TITLE
POST new product

### PR DESCRIPTION
**Pull request by kachmashk**

### CODE REVIEW

File: src/components/products/components/modalCreate/ModalCreateProduct.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+import Col from 'react-bootstrap/Col'
+import Button from 'react-bootstrap/Button'
+import ProductsApiService from '../../../../utils/api/productsApiService'
+import './ModalCreateProduct.scss'
+
+export default function ProductCreateModal(props) {
+	const [product, setProduct] = useState({})
+	const [isValidated, setIsValidated] = useState(false)
+
+	const title = props.title ? props.title : 'Create a Product Modal'
```
Comment (bionanek) [inline src/components/products/components/modalCreate/ModalCreateProduct.js]: change default title to 'Create Product'

File: src/components/products/components/modalCreate/ModalCreateProduct.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+	const handleModalClose = () => {
+		props.onModalClose()
+	}
+
+	const handleAddCurrency = (currentProduct, currency, value) => {
```
Comment (bionanek) [inline src/components/products/components/modalCreate/ModalCreateProduct.js]: maybe rename to something like 'updateProductPrice'

File: src/components/products/components/modalCreate/ModalCreateProduct.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+			editableProduct.price = {
+				[currency]: value,
+			}
+		} else {
+			editableProduct.price[currency] = value
+		}
+		return editableProduct
+	}
+
+	const handleChange = event => {
```
Comment (bionanek) [inline src/components/products/components/modalCreate/ModalCreateProduct.js]: rename to handleFormControlChange

File: src/components/products/list/ProductsList.js [hunk 4]
```diff
@@ -53,18 +64,48 @@ class ProductsList extends Component {
 		this.setState({ products })
```
Comment (bionanek) [inline src/components/products/list/ProductsList.js]: I think that after you added isProductCreateModalOpen to the state, now whenever you call setState() you might need to specify the name of the property you are updating.... So in here for example I think it should be changed to `this.setState({products: products})`

Comment (bionanek) [reply src/components/products/list/ProductsList.js]: just like you did in openProductModal() or closeProductModal()

File: src/components/products/list/ProductsList.js [hunk 5]
```diff
@@ -53,18 +64,48 @@ class ProductsList extends Component {
...
-					clickable
+			<>
+				<Row>
+					<Col>
+						<Button
+							className="button-create-product float-right"
+							variant="primary"
+							onClick={this.openProductModal}
+						>
+							Create a Product
```
Comment (bionanek) [inline src/components/products/list/ProductsList.js]: maybe rename to Add Product, feels better

File: src/components/products/list/ProductsList.js (hunk 4 above)
Comment (kachmashk) [reply src/components/products/list/ProductsList.js]: Can be for clarity, just ESLint is saying that 'Expected property shorthand.'

**Pull request merged**